src/App.tsx

```diff
@@ -22,6 +22,7 @@ function App() {
             <Route path="/series" element={<SeriesPage />} />
             <Route path="/animation" element={<AnimationPage />} />
           </Routes>
+        <Footer />
```

Comment:
🔧 If the footer is going to persist across the application and never change, it's a great candidate for `useMemo`